docs/guides/volume-populator/pvc/index.md

```diff
@@ -0,0 +1,451 @@
+---
+title: Populate Volume of a Stand-alone PVC | KubeStash
+description: An step by step guide to show how to populate volume of a stand-alone PVC
+menu:
+  docs_{{ .version }}:
+    identifier: volume-populate-pvc
+    name: Populate Volume of a Stand-alone PVC
+    parent: volume-populator
+    weight: 20
+product_name: kubestash
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+# Populating a volume of a Standalone PVC
+
+This guide will show you how to use KubeStash to populate volume of a Stand-alone PersistentVolumeClaims (PVCs). Within this guide, we will demonstrate the process of backing up a Stand-alone `PVC` and subsequently restoring it using Kubernetes native approach, facilitated by KubeStash.
+
+## Before You Begin
+
+- At first, you need to have a Kubernetes cluster, and the `kubectl` command-line tool must be configured to communicate with your cluster. If you do not already have a cluster, you can create one by using [kind](https://kind.sigs.k8s.io/docs/user/quick-start/).
+- Install `KubeStash` in your cluster following the steps [here](/docs/setup/README.md).
+- You should be familiar with the following `KubeStash` concepts:
+    - [BackupStorage](/docs/concepts/crds/backupstorage/index.md)
+    - [BackupConfiguration](/docs/concepts/crds/backupconfiguration/index.md)
+    - [BackupSession](/docs/concepts/crds/backupsession/index.md)
+    - [SnapShot](/docs/concepts/crds/snapshot/index.md)
+    - [RestoreSession](/docs/concepts/crds/restoresession/index.md)
+    - [RetentionPolicy](/docs/concepts/crds/retentionpolicy/index.md)
+
+To keep everything isolated, we are going to use a separate namespace called `demo` throughout this tutorial.
+
+```bash
+$ kubectl create ns demo
+namespace/demo created
+```
+
+> **Note:** YAML files used in this tutorial are stored in [docs/guides/volumes/pvc/examples](/docs/guides/volumes/pvc/examples) directory of [kubestash/docs](https://github.com/kubestash/docs) repository.
+
+## Prepare Volume
+
+At first, let's prepare our desired PVC. Here, we are going to create a PVC. Then, we are going to mount this PVC with a Pod. Pod will generate a sample file into that PVC.
+
+**Create PersistentVolumeClaim:**
+
+Below, is the YAML of the PVC that we are going to create,
+
+```yaml
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: target-pvc
+  namespace: demo
+spec:
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 1Gi
+```
+
+Notice the `spec.accessModes` section. We are using `ReadWriteMany` access mode so that multiple pods can use this PVC simultaneously. Without this access mode, KubeStash will fail to backup the volume if any other pod mount it during backup.
```

Comment:
`ReadWriteMany` should not be necessary. For workload backup, we schedule the backup pods in the same node. For PVC backup, we can specify the node name from the `jobTemplate`.